src/TMU.cpp

```diff
@@ -13,9 +13,16 @@ uint16_t TMU::getObjectDictionarySize() const {
     return OBJECT_DICTIONARY_SIZE;
 }
 
-void TMU::updateTemps() {
+void TMU::process() {
+    uint16_t thermTemps[NUM_THERMOCOUPLES] = {};
+
     for (int i = 0; i < 4; i++) {
```

Comment:
This doesn't make a huge difference, but it is a bit more consistent this way
```suggestion
    for (int i = 0; i < NUM_THERMOCOUPLES; i++) {
```